home/templates/admin/team_formation.html

```diff
@@ -0,0 +1,789 @@
+                                    <tr>
+                                        <th>{% trans "Name" %}</th>
+                                        <th>{% trans "Score" %}</th>
+                                        <th>{% trans "Rank" %}</th>
+                                        <th title="{% trans 'Hours of overlap with captain' %}">{% trans "Capt. Hrs" %}</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    {% for djangonaut_data in team_data.djangonaut_details %}
+                                    <tr>
+                                        <td>{{ djangonaut_data.user.get_full_name|default:djangonaut_data.user.email }}</td>
+                                        <td>{{ djangonaut_data.score|default:"—" }}</td>
+                                        <td>{{ djangonaut_data.selection_rank|default:"—" }}</td>
+                                        <td>
+                                            {% if djangonaut_data.captain_hours is not None %}
+                                                {{ djangonaut_data.captain_hours }}
+                                                {% if djangonaut_data.captain_hours < 2 %}
+                                                    <span style="color: #dc3545;">⚠</span>
+                                                {% else %}
+                                                    <span style="color: #28a745;">✓</span>
+                                                {% endif %}
+                                            {% else %}
+                                                <span style="color: #999;">—</span>
+                                            {% endif %}
+                                        </td>
+                                    </tr>
+                                    {% endfor %}
+                                </tbody>
+                            </table>
+                            {% else %}
+                                <p style="font-size: 0.85em; color: #666; margin: 0.5rem 0;">
+                                    <em>{% trans "No djangonauts assigned" %}</em>
+                                </p>
+                            {% endif %}
+                        </div>
+                    </div>
+                    {% endfor %}
+                {% else %}
+                    <p style="text-align: center; color: #666; padding: 2rem;">
+                        <em>{% trans "No teams created yet." %}</em>
+                    </p>
+                {% endif %}
+            </div>
+        </aside>
+
+    </div>
+</div>
+
+<script>
+// Sortable columns
+document.querySelectorAll('.sortable').forEach(th => {
+    th.addEventListener('click', function() {
+        const sortBy = this.getAttribute('data-sort');
+        const currentSort = '{{ sort_by }}';
+        const currentOrder = '{{ sort_order }}';
```

Comment:
Move this into a `{{ sort_options|json_object }}` pattern to avoid tying this javascript to the django template rendering.